src/redux/category/category.js

```diff
@@ -1,16 +0,0 @@
-const CHECK_STATUS = 'CHECK_STATUS';
-
-const categoryReducer = (state = [], action) => {
-  switch (action.type) {
-    case CHECK_STATUS:
-      return 'UNDER CONSTRUCTION';
-    default:
-      return state;
-  }
-};
-
-export const checkStatus = () => ({
-  type: CHECK_STATUS,
-});
-
-export default categoryReducer;
```

Comment:
- [ ] the categories file is empty right now please make sure to fix and revert the category logic in order to match the requirements